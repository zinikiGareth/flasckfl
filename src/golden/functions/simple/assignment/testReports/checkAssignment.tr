PASS that x has the right value
FAIL that we can have failing tests
