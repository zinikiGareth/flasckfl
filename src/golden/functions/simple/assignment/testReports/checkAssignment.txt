PASSED: that x has the right value
PASSED: that x has the right value
